### PR TITLE
chore(Carta della cultura): [AP-15] Carta della cultura networking

### PR DESCRIPTION
**Pull request by debiff**
## Short description
This PR adds the networking layer for the management of the carta della cultura API.

## List of changes proposed in this pull request
- Added the cdc client 
- Added the api handler saga

## How to test
Checkout the branch of this PR in dev-server: https://github.com/pagopa/io-dev-api-server/pull/167
Try to dispatch the `cdcRequestBonusList.request` and the `cdcEnrollUserToBonus.request` actions.

If you want you can use the Reactotron utils as follow:
```
RTron.onCustomCommand({
    command: "BonusList",
    handler: () =>
      dispatch(cdcEnrollUserToBonus.request()),
    title: "BonusList" 
  });
RTron.onCustomCommand({
    command: "failure",
    handler: () =>
      dispatch(cdcEnrollUserToBonus.request([{ year: "2020" as Anno }])),

    // Optional settings
    title: "Dispatch failure" // This shows on the button
  });
```

### CODE REVIEW

File: ts/features/bonus/cdc/saga/networking/handlePostRegistraBeneficiario.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      if (postRegistraBeneficiarioResult.value.status === 200) {
+        yield* put(
+          cdcEnrollUserToBonus.success(
+            convertSuccess(postRegistraBeneficiarioResult.value.value)
+          )
+        );
+        return;
+      }
+      if (postRegistraBeneficiarioResult.value.status === 400) {
+        cdcEnrollUserToBonus.failure(
```
Comment (emiliopavia) [inline ts/features/bonus/cdc/saga/networking/handlePostRegistraBeneficiario.ts:46]: Same as above (`yield* put`).

File: ts/features/bonus/cdc/saga/networking/handlePostRegistraBeneficiario.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      yield* put(
+        cdcEnrollUserToBonus.failure(
+          getGenericError(
+            new Error(postRegistraBeneficiarioResult.value.status.toString())
+          )
+        )
+      );
+      return;
+    }
+    cdcEnrollUserToBonus.failure(
```
Comment (emiliopavia) [inline ts/features/bonus/cdc/saga/networking/handlePostRegistraBeneficiario.ts:62]: Same as above (`yield* put`).

**Review by emiliopavia — CHANGES_REQUESTED** [review]

File: ts/features/bonus/cdc/saga/networking/handleGetStatoBeneficiario.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      yield* put(
+        cdcRequestBonusList.failure(
+          getGenericError(
+            new Error(getStatoBeneficiarioResult.value.status.toString())
+          )
+        )
+      );
+      return;
+    }
+    cdcRequestBonusList.failure(
```
[the remark this answers is not in the document]
Comment (debiff) [reply ts/features/bonus/cdc/saga/networking/handleGetStatoBeneficiario.ts]: Sure, thank you! Fixed in [cc3e480](https://github.com/pagopa/io-app/pull/3927/commits/cc3e48002f56fce8b0a029df9e3656518b0efca4)

File: ts/features/bonus/cdc/saga/networking/handlePostRegistraBeneficiario.ts (hunk 1 above)
Comment (debiff) [reply ts/features/bonus/cdc/saga/networking/handlePostRegistraBeneficiario.ts]: Fixed in [cc3e480](https://github.com/pagopa/io-app/pull/3927/commits/cc3e48002f56fce8b0a029df9e3656518b0efca4)

File: ts/features/bonus/cdc/saga/networking/handlePostRegistraBeneficiario.ts (hunk 2 above)
Comment (debiff) [reply ts/features/bonus/cdc/saga/networking/handlePostRegistraBeneficiario.ts]: Fixed in [cc3e480](https://github.com/pagopa/io-app/pull/3927/commits/cc3e48002f56fce8b0a029df9e3656518b0efca4)

File: assets/CdcSwagger.yml [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+        - application/json
+      produces:
+        - application/json
+swagger: '2.0'
+basePath: /$%7Bhost%7D
+definitions:
+  StatoBeneficiario:
+    type: string
+    enum:
+      - ATTVABILE
```
Comment (emiliopavia) [inline assets/CdcSwagger.yml:69]: There's a typo here, maybe you wanna fix it before merging.

**Review by emiliopavia — APPROVED** [review]

File: assets/CdcSwagger.yml (hunk 4 above)
Comment (emiliopavia) [reply assets/CdcSwagger.yml:69]: Absolutely 👍

**Pull request merged**